source/material_model/grain_size.cc

```diff
@@ -1064,6 +1109,12 @@ namespace aspect
                              "The grain size $d_{ph}$ to that a phase will be reduced to when crossing a phase transition. "
                              "When set to zero, grain size will not be reduced. "
                              "Units: \\si{\\meter}.");
+          prm.declare_entry ("Volume fraction of phase", "0.4",
+                             Patterns::List (Patterns::Double (0., 1.)),
+                             "The volume fraction of one of the phases in the two phases evolved using Bercovici and "
+                             "Richard (2012). The volume fraction of the other phase can be simply calculated by subtracting "
```

Comment:
```suggestion
                             "Ricard (2012). The volume fraction of the other phase can be simply calculated by subtracting "
```
I think this might also be confusing, since everywhere else we mention the Mulyokova paper. So if the user is not aware that they use this damage model, they might not know that this parameter is only used when we use the pinned damage model. I think this would be worth adding.